src/index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>learderbord</title>
+</head>
+<body>
+<header>
+  <h1 class="header">learderbord</h1>
+</header>
+  <div class="main-container d-flex">
+  <section class="first-part">
+    <div class="score-header d-flex">
+        <h2>recent scores</h2>
+        <button type="submit">refresh</button>
+    </div>
+      <div class="ul">
+          <ul class="scores">
+              <li>name:500</li>
+              <li>name:500</li>
+              <li>name:500</li>
+              <li>name:500</li>
+              <li>name:500</li>
+              <li>name:500</li>
+              <li>name:500</li>
+              <li>name:500</li>
+              <li>name:500</li>
+              <li>name:500</li>
+          </ul>
+        </div>
```

Comment:
- [ ] good job creating the project and making it look like the wireframe 👍 however right now the code you wrote is completely in HTML the goal of the assignment is to make use of ES6 modules to write modular JavaScript with proper ES6 syntax. please create the list items using ES6 arrow functions. Please refer to [this article](https://www.w3schools.com/js/js_arrow_function.asp) for more information regarding arrow functions.

Reply:
This issue has been addressed well! Good job! :clap: :clap: :clap:

Reply:
@mikethreels thank you for your review